README.md

```diff
@@ -117,7 +117,7 @@ Output consumed messages in JSON envelope:
 
 Output consumed messages according to format string:
 
-    $ kafkacat -b mybroker -t syslog -f 'Topic %t[%p], offset: %o, key: %k, payload: %S bytes: %s\n'
+    $ kafkacat -b mybroker -t syslog -f 'Topic %t[%p], offset: %o, key: %k, payload: %s bytes: %S\n'
```

Comment:
This will print something like:
`.. payload: The Actual Payload bytes: 23`, which is not what we want, we want it to say:
`.. payload: 23 bytes: The Actual Payload`, which the existing example does, right?